### PR TITLE
Update Project by Guid Data Request

### PR DESCRIPTION
**Pull request by MackMcCall**
- Created UpdateProjectByGuid data request
- Handled null for name and null or empty for isActive
- Created tests

### CODE REVIEW

**Review by Matthew-Brogan — APPROVED** [review]

**Review by ErikHedman23 — COMMENTED** [review]
Looks good!  Great work Mack!

**Review by ErikHedman23 — APPROVED** [review]

File: TicketMate.Persistence/DataRequestObjects/ProjectRequests/UpdateProjectByGuid.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+        public string? Name { get; set; }
+
+        public bool? IsActive { get; set; }
+
+        public UpdateProjectByGuid(Guid guid, string? name, bool? isActive) 
+        {
+            Guid = guid;
+
+            if (name == "") name = null;
```
Comment (uhyeay2) [inline TicketMate.Persistence/DataRequestObjects/ProjectRequests/UpdateProjectByGuid.cs:21]: can we set name to null if string.IsNullOrWhiteSpace(name) ?

**Review by uhyeay2 — APPROVED** [review]

File: TicketMate.Persistence/DataRequestObjects/ProjectRequests/UpdateProjectByGuid.cs (hunk 1 above)
Comment (MackMcCall) [reply TicketMate.Persistence/DataRequestObjects/ProjectRequests/UpdateProjectByGuid.cs:21]: For sure! I'd love to hear more on this today at standup.

**Review by BraydonTGS — APPROVED** [review]
Great work!

**Pull request merged**